### PR TITLE
docs: update homepage

### PR DESCRIPTION
**Pull request by clemcode**

### CODE REVIEW

File: docs/components/HomeFeatures.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      {
+        title: 'Resize and transform',
+        description:
+          'Generate images at the right size, defined in your components or following your breakpoints. Change the format and quality of your images.',
+        icon: 'IconResize'
+      },
+      {
+        title: 'Image providers',
+        description:
+          'Use your favorite image providers with ease or use the built-in one. Imgix, Cloudinary, Prismic, Twicpics, Vercel and more are supported by default.',
```
Comment (pi0) [inline docs/components/HomeFeatures.vue:44]: We should avoid marketing vendor names.

File: docs/components/HomeFeatures.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        icon: 'IconSparkles'
+      },
+      {
+        title: 'Standards compliant',
+        description:
+          'Our components generate standard &lt;img&gt; and &lt;picture&gt; tags. No tricky wrapper so you can style your images with confidence.',
```
Comment (pi0) [inline docs/components/HomeFeatures.vue:50]:
```suggestion
          'Image components generate standard &lt;img&gt; and &lt;picture&gt; tags. No tricky wrapper so you can style your images with confidence.',
```

File: docs/components/HomeFeatures.vue (hunk 1 above)
Comment (pi0) [reply docs/components/HomeFeatures.vue:44]:
```suggestion
          'Choose from 10+ ready to use providers or use the built-in optimizer. ',
```

File: docs/components/HomeFeatures.vue [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+<script>
+import { defineComponent } from '@nuxtjs/composition-api'
+
+export default defineComponent({
+  setup() {
+    const features = [
+      {
+        title: 'Resize and transform',
+        description:
+          'Generate images at the right size, defined in your components or following your breakpoints. Change the format and quality of your images.',
```
Comment (pi0) [inline docs/components/HomeFeatures.vue:38]:
```suggestion
          'Generate images at the right size, defined by component or following responsive breakpoints. Use modern formats and adjust quality of website images.',
```

File: docs/components/HomeHero.vue [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                items-center
+                justify-center
+                font-medium
+                px-4
+                py-3
+                sm:px-8
+                text-white
+                rounded
+                hover:border
+              "
```
Comment (danielroe) [inline docs/components/HomeHero.vue:91]: Might be nice to have a transition on this and 'Get started'

File: docs/components/HomeHero.vue [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+                px-4
+                py-3
+                border border-transparent
+                text-base
+                font-medium
+                rounded-md
+                shadow-sm
+                text-white
+                bg-primary-500
+                sm:px-8
```
Comment (danielroe) [inline docs/components/HomeHero.vue:72]: Could we have a hover/focus/active state?

**Review by danielroe — COMMENTED** [review]
Looks amazing! 😍

File: docs/components/HomeHero.vue [hunk 6]
```diff
@@ -0,0 +1,100 @@
+<template>
+  <main class="d-container">
+    <!-- Hero card -->
+    <div class="relative sm:rounded-2xl sm:overflow-hidden">
+      <!-- Background -->
+      <div class="absolute inset-0">
+        <img
```
Comment (pi0) [inline docs/components/HomeHero.vue:7]: There is flicker in background loading not sure related to this or not

File: docs/components/HomeFeatures.vue (hunk 1 above)
Comment (Atinux) [reply docs/components/HomeFeatures.vue:44]: Why not but we need something longer to be balanced with other features.

Screenshot 2021-05-27 at 18 44 16@2x

Comment (pi0) [reply docs/components/HomeFeatures.vue:44]: Since is not fair to others nuxt is agnostic 😸  Updated suggestion to be longer.

Comment (Atinux) [reply docs/components/HomeFeatures.vue:44]: What other sentence can we add @pi0 @danielroe?

File: docs/components/HomeHero.vue (hunk 4 above)
Comment (Atinux) [reply docs/components/HomeHero.vue:91]: What transition do you have in mind?

Comment (danielroe) [reply docs/components/HomeHero.vue:91]:
```suggestion
              class="
                flex
                items-center
                justify-center
                font-medium
                px-4
                py-3
                transition
                duration-300
                sm:px-8
                text-white
                rounded
                ring-white hover:ring-2
              "
```

File: docs/components/HomeHero.vue (hunk 5 above)
Comment (danielroe) [reply docs/components/HomeHero.vue:72]:
```suggestion
              class="
                flex
                items-center
                justify-center
                px-4
                py-3
                transition
                duration-300
                hover:bg-primary-600
                ring-white focus:ring-2 ring-offset-2 ring-offset-primary-800
                border border-transparent
                text-base
                font-medium
                rounded-md
                shadow-sm
                text-white
                bg-primary-500
                sm:px-8
```

File: docs/components/HomeHero.vue (hunk 6 above)
Comment (clemcode) [reply docs/components/HomeHero.vue:7]: There was a mistake with the initial SVG export, I think it will fix the problem.

**Comment by pi0** [comment]
There is also a hydration error when refreshing docs for nav

<a href="https://www.loom.com/share/d263b1a26ff84471ac753cb80ce9aed9"> <p>Google Chrome - Optimised images for Nuxt - Google Chrome - Watch Video</p>  </a>

**Comment by Tahul** [comment]
Hello @pi0 ; should be fixed in latest commits 😄

File: docs/components/HomeHero.vue [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+                text-white
+                bg-primary-500
+                sm:px-8
+              "
+            >
+              Get started
+            </nuxt-link>
+
+            <a
+              href="https://githubbox.com/nuxt/image/tree/main/example"
```
Comment (pi0) [inline docs/components/HomeHero.vue:78]:
```suggestion
              href="https://githubbox.com/nuxt/image/tree/main/example"
              target="_blank"
              rel="noopener"
```

File: docs/components/HomeFeatures.vue (hunk 1 above)
Comment (danielroe) [reply docs/components/HomeFeatures.vue:44]:
```suggestion
          'Choose from 10+ ready-to-use popular providers, add your own custom provider in a few lines of code - or use the default settings to generate automatically optimized static images.',
```

**Comment by pi0** [comment]
Awesome work @Tahul @clemcode <3

**Review by pi0 — APPROVED** [review]

**Comment by pi0** [comment]
@Atinux It is same in `main` (as well as some other small issues). Will directly patch.

**Comment by clemcode** [comment]
We're almost there, finishing the video, I let you merge the PR as soon as you're good with it !

**Pull request merged**